importer/src/path_canonicalizer.rs

```diff
@@ -255,8 +255,6 @@ fn canonicalize_inside_expression(
                 // If resolving the reference fails, we assume it is a local variable that has been checked below.
                 if let Some(n) = paths.get(&path.clone().join(reference.path.clone())) {
                     *reference = n.relative_to(&Default::default()).into();
-                } else {
-                    assert!(reference.path.try_to_identifier().is_some());
```

Comment:
@chriseth we talked about the real issue here being that the first step of canonicalization did not run. It is not, the first step does run. Also, I checked with `instr incr_a { A = utils::super::utils::incremented(A) }` and it gets resolved correctly (with this assertion removed).

Reply:
looking into this